server/routes/search/searchController.ts

```diff
@@ -115,10 +118,31 @@ export default class SearchController {
     return res.render('pages/suggestion')
   }
 
-  getPaginationParams(req: Request, page: PageMetaData): LegacyPagination {
-    // const filters = req.session.searchParams.filters ? `&filters=${req.session.searchParams.filters}` : ''
-    // const paginationUrlPrefix = `/search/results?${filters}&`
-    const paginationUrlPrefix = `/search/results?`
+  getSessionFilterString(req: Request): string {
+    const searchParams = new URLSearchParams()
```

Comment:
I am doing a lot of faffing with the filter string - will see if I can tidy this up a bit